packages/stg-devtools-evm-hardhat-v2/src/asset/sdk.ts

```diff
@@ -47,7 +47,8 @@ export class Asset extends Ownable implements IAsset {
 
         return {
             ...this.createTransaction(data),
-            description: `Setting address config for ${formatOmniPoint(omniContractToPoint(this.contract))}:\n${printRecord(config)}`,
+            description: `Setting address config for ${formatOmniPoint(omniContractToPoint(this.contract))}`,
+            metadata: { functionName: 'setAddressConfig', functionArgs: `config = ${printRecord(config)}` },
```

Comment:
This is currently throwing an error because I need to update the devtools dependency once the [devtools PR ](https://github.com/LayerZero-Labs/devtools/pull/991/files)is merged and published